discovery_engine_core/web-api2/tests/cmd/cli_overrides.auto.toml

```diff
@@ -0,0 +1,36 @@
+bin.name = "ingestion2"
+args = ["--print-config", "--config", "tests/cmd/assets/config.toml", "--bind-to", "127.4.3.2:1099"]
+status.code = 0
+stderr = ""
+stdout = """
+{
+  "logging": {
+    "file": "foo/bar.log"
+  },
+  "net": {
+    "bind_to": "127.4.3.2:1099",
+    "max_body_size": 64
+  },
+  "elastic": {
+    "url": "http://localhost:3219",
+    "user": "tic",
+    "password": "[REDACTED]",
+    "index_name": "other_index"
+  },
+  "db": {
+    "base_url": "postgres://user:pw@localhost:5432/xayn",
```

Comment:
should we maybe hide from stdout the password in the url too?

Reply:
I hadn't considered it as I think any password (besides the hard coded one for testing) should never be in the url :thinking: 

I wouldn't want to hide the whole url as it is usefull for debugging but I guess we can hide part of it :thinking: